backend/tests/test_engine_plugin.cpp

```diff
@@ -29,10 +35,11 @@ TEST(GPU_EnginePluginTest, LoadPluginsAndExecuteOpGraph)
     Test_engine_plugin_manager plugin_manager;
 
     // Create a list of paths to plugins
-    std::vector<std::filesystem::path> plugin_paths = {"./hipdnn_test_engine_plugin1"};
+    std::vector<std::filesystem::path> plugin_paths
+        = {"./libhipdnn_test_engine_plugin1" + std::string(SHARED_LIB_EXT)};
```

Comment:
The shared object name is `libhipdnn_test_engine_plugin1.so` for Windows and `hipdnn_test_engine_plugin1.dll` for windows. `Shared_object` handles this internally.